arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -951,6 +951,12 @@ def test_plot_posterior(models, kwargs):
         assert axes.shape
 
 
+def test_plot_posterior_boolean():
+    data = np.random.choice(a=[False, True], size=(4, 100))
+    axes = plot_posterior(data)
+    assert axes
```

Comment:
can you also add an assert with `get_xticklabels` or something like that to make sure they are not 0 and 1? We don't need to check every aspect of the plot, I propose this one because I think it's the easiest one and will catch breaking changes if we update the plot and change the labels for example

Reply:
`axes.get_xticklabels()` returns 
`[Text(0, 0, 'False'), Text(1, 0, 'True')]` in boolean case

Not very sure how to tackle this one. It needs some attention

Reply:
Maybe 
`assert axes.get_xticklabels()[0].get_text()=='False'` this will do the job?